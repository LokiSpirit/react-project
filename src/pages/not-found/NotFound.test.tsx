import { render } from '@testing-library/react';
import NotFound from './NotFound';

describe('NotFound component', () => {
  it('renders the component correctly', () => {
    const { getByText } = render(<NotFound />);
    const notFoundElement = getByText('404 - Not Found');
    expect(notFoundElement).toBeInTheDocument();
  });

  it('matches snapshot', () => {
    const { asFragment } = render(<NotFound />);
    expect(asFragment()).toMatchSnapshot();
  });
});
