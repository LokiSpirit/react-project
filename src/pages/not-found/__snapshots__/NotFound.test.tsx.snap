// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`NotFound component > matches snapshot 1`] = `
<DocumentFragment>
  <div>
    404 - Not Found
  </div>
</DocumentFragment>
`;
